app/views/header.scala.html

```diff
@@ -15,7 +15,7 @@
                 <a class="nav-link" href="/gallery">@messages("header.gallery")</a>
             </li>
             <li class="nav-item">
-            @if(loggedIn) {
+            @if(user.length>2) {
```

Comment:
`user.length>2` I don't understand the logic here

Reply:
when creating an account, the minimum length of a valid username is 3 chars so >2 accepts valid users

Reply:
Also allows me to pass user as a string for both validations and to display user name